File: Solutions/242. Valid Anagram/242.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] set = new int[26];
        int m = s.length(), n = t.length();
        if(m!=n){
            return false;
        }
        for(int i=0;i<m;i++){
            set[s.charAt(i) - 'a']++;
        }
        for(int i=0;i<n;i++){
            set[t.charAt(i) - 'a']--;
        }
        for(int i=0;i<26;i++){
            if(set[i] != 0){
                return false;
            }
        }
        return true;
    }
}

File: Solutions/242. Valid Anagram/242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = defaultdict(int)
        for i in s:
            count[i] += 1
        for i in t:
            count[i] -= 1
        for i in count.values():
            if i != 0:
                return False
        return True